class.jetpack-jitm.php

```diff
@@ -91,7 +91,7 @@ static function jitm_woocommerce_services_msg( $content ) {
 
 		$base_location = wc_get_base_location();
 
-		switch ( $base_location['country'] ) {
+		switch ( $base_location[ 'country' ] ) {
```

Comment:
Let's not change that - it's against [WordPress coding spacing standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage) for PHP. There are a lot of other examples of this change in this PR, can you please revert them?